leptos_dom/src/components/dyn_child.rs

```diff
@@ -176,7 +176,7 @@ where
             let child = component.child.clone();
 
             #[cfg(all(
-                debug_assertions,
+                any(debug_assertions, feature="ssr"),
```

Comment:
This one will not work as intended because it has additional checks for wasm32/web on the next lines -- should be `#[cfg(any(feature = "ssr", all(debug_assertions, target_arch = "wasm32", feature = "web")))]` or something, or just drop the part about wasm32/web